packages/devextreme/js/ui/tag_box.js

```diff
@@ -1485,6 +1485,10 @@ const TagBox = SelectBox.inherit({
         return existedItems.concat(newItems);
     },
 
+    _isCustomItemSelected() {
```

Comment:
Let's think a bit more how can we rework the actual algorithm to fix the issue.
Function with name `_isCustomItemSelected` should not return just `acceptCustomValue`. Moreover, as far as i see we already have the check for `acceptCustomValue` in `valueChangeHandler`. It would be great if we research why we need this code in selectBox and prepare more intuitive solution.

Reply:
fix reworked